### ISSUE TITLE
Criar tabela simples de visualização de descritores por nível

### ISSUE THREAD

**Issue by gBritz** [Famintus/WebPlataform#340]
Substituir a funcionalidade atual de edição de descritores por uma tabela simples de listagem dos descritores associados aos níveis. 
A tabela é igual à tabela atual, mas é apenas para visualização, não permite nenhuma edição. A edição dos níveis vai ser feita somente pela função Seed.

Acrescentar na descrição do nível o número que está armazenado em "Order ID" separado por " - ".
Ex: "3 - Separação de sílabas"

Utilizar tabela Knockout padrão de demais tabelas de visualização do sistema.

Utilizar o mesmo submenu "Descritores".

**Comment by gBritz** [comment]
**Status in Jira**
    Fabio Beckenkamp (fabio@elefanteletrado.com.br) `Created` at fabio@elefanteletrado.com.br

    Fabio Beckenkamp (fabio@elefanteletrado.com.br) set `In Progress` at 11/12/2014 11:40:28
    Fabio Beckenkamp (fabio@elefanteletrado.com.br) set `Closed` at 11/12/2014 11:40:28

**Comment by gBritz** [comment]
**Commentary** by Raphael Vilvert (rvilvert@outlook.com)
as Issues YAA-85 e 86 afetam o mesmo controller, por isto vou finalizar as issues juntas.

**Comment by gBritz** [comment]
**Commentary** by Fabio Beckenkamp (fabio@elefanteletrado.com.br)
Sugestão de melhorias:

- Congelar as duas primeiras colunas: Habilidades e Descritores, assim é possível navegar melhor nas últimas colunas de níveis.
- Ordenação dos níveis é somente ascendente (mostrando as linhas marcadas primeiro)
- Pode ser considerado colocar um valor de número de itens por página também. Assim pode-se listar todos itens de uma só vez se for desejado.

**Comment by gBritz** [comment]
**Commentary** by Raphael Vilvert (rvilvert@outlook.com)
Página alterada.

* Removido arquivos que não são mais necessários na área admin: 
** DescriptorNew.js
** Skillset.js 
** Skillset.cshtml

* Melhoria na partial view do Pager para fazer paginação pela Model também, assim não é necessário criar uma Session para os dados uma vez que a requisição é sempre feita no servidor e o session é reconstruído.

* Adicionado novo Helper customizado para criação de menu na área adm baseado no perfil para evitar duplicar códigos html, para utilizar basta usar Html.AdminMenuItem(sysMenuName, ActionLink, ControllerName, itemTitle). Agora também não precisa mais colocar a regra da Role do menu na View, ela já é verificada no AdminMenuItem

* Adicionado nova partial view compartilhada entre as views de Skills e Descriptor (_Menu) utilizando o novo Helper.

* Refactor geral na SkillController removendo os métodos não utilizados e reescrevendo os métodos de busca das Skills e Descriptors. 

* Adicionado novo Package no projeto WEB: Linq.Dynamic https://github.com/kahanu/System.Linq.Dynamic O Dynamic foi adicionado durante o refactor da SkillController para permitir a paginação dinâmica sem a necessidade de adicionar um IF para cada coluna possível. 

* Para a verificação dos métodos e arquivos excluídos foi utilizado a busca por referência do Visual Studio e depois o AstroGrep para buscar o nome fisicamente em todos os arquivos da solução.

* As ViewModels de Descritores e Habilidades também foram revisadas.

* O knockout não foi utilizado nas outras tabelas de visualização do sistema, conversei com o Diogo e foi utilizado apenas nas páginas de formulários. Minha sugestão é criar um Helper de Grid com base na Model utilizando knockout. Hoje as tabelas são Listas simples sem requisição Ajax.

**Comment by gBritz** [comment]
**Commentary** by Raphael Vilvert (rvilvert@outlook.com)
Vou abrir uma sub tarefa pro Natã verificar a questão de congelar as colunas. Sobre os outros 2 itens estou tratando na outra issue .
